include/nccl-headers/nvidia/net_device.h

```diff
@@ -14,7 +14,7 @@
 
 // Arbitrary version number - A given NCCL build will only be compatible with a single device networking plugin
 // version. NCCL will check the supplied version number from net->getProperties() and compare to its internal version.
-#define NCCL_NET_DEVICE_UNPACK_VERSION 0x7  
```

Comment:
nack.  Please leave the headers in `include/nccl-headers/nvidia` as *exact* copies of what's in the NCCL repo.